instrumentation/http_instrumentation.go

```diff
@@ -19,36 +24,36 @@ type httpMetricsCollector struct {
 }
 
 // newHTTPMetricsCollector creates a new httpMetricsCollector.
-func newHTTPMetricsCollector(reg *prometheus.Registry, hardcodedLabels []string) httpMetricsCollector {
+func newHTTPMetricsCollector(reg *prometheus.Registry) httpMetricsCollector {
```

Comment:
Maybe let's not remove this functionality and break API? For any other users of this package, `hardcodedLabels` would have made a lot of sense and they can instrument different things as they want to.

Reply:
I fixed all usages of this package.

Reply:
Plus this is an unexported function.